src/pages/patientView/mutation/column/AlleleFreqColumnFormatter.tsx

```diff
@@ -161,18 +153,37 @@ export default class AlleleFreqColumnFormatter {
 
 
     public static getFrequency(data:Mutation[]): string|string[] {
```

Comment:
Should we name this function `download` for consistency?

Reply:
No objections, go ahead to change the name if it is more consistent with the code.